vtipnik.py

```diff
@@ -18,16 +19,24 @@ def seznam_vtipu():
     """
     Funkce načtě obsah složky a vrátí seznam všech dostupných vtipů
     """
-    return glob("vtipy/*")
+    return glob.glob("vtipy/*")
 
 
 def main():
     """
-    Vypíše všechny nalezené vtipy.
+    Vypíše jeden náhodný vtip ze seznamu a po stisknutí Enteru vypíše další
     """
-    for vtip in seznam_vtipu():
-        print("=" * 80)
-        print(nacti_vtip(vtip))
+    ls_vtip=seznam_vtipu()
+    for x in range(len(ls_vtip)):
+       print("=" * 80) 
+       vtip=random.choice(ls_vtip)
+       print(nacti_vtip(vtip))
+       ls_vtip.remove(vtip)
+       if len(ls_vtip)==0:
```

Comment:
Tady stačí jen `if not ls_vtip:`, prázdný seznam == False, neprázdný == True. Ovšem je tohle vůbec potřeba? Neskončí for cyklus dříve než se vyprázdní seznam a my se dostaneme k této podmínce?

Reply:
Uznávám že zapsání toho cyklu je trohu nesmyslné ješte to upravím